Automattic/pocket-casts-ios
# [Deselecting Chapters] Persistence

| 📘 Part of: #1396 | Depends on #1426 |
|:---:|:---:|

Persists users' deselected chapters on the database.

These are podcasts with chapters you can use to test:

* Clublife: https://pca.st/rLzZCv
* ATP: https://pca.st/accidentaltech

## To test

2. Run the app
3. Play any episode with chapters
4. Deselect some chapters
5. Close and reopen the app
6. ✅ Your chapter list should reflect your previous selection
7. Toggle between episode with chapters and change their selections
8. ✅ Your chapter selection should be correctly reflected

## Checklist
